libecole/include/ecole/environment/cutting.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "ecole/dynamics/cutting.hpp"
+#include "ecole/environment/environment.hpp"
+#include "ecole/information/nothing.hpp"
+#include "ecole/observation/node-bipartite.hpp"
+#include "ecole/reward/is-done.hpp"
+
+namespace ecole::environment {
+
+template <
+	typename ObservationFunction = observation::NodeBipartite,
```

Comment:
Not sure this will be the best default observation function for this, but we'll see later.